### ISSUE TITLE
YELLOW FILE DIALOG CAN'T BE DISMISSED

### ISSUE THREAD

**Issue by dbareis** [notepad-plus-plus/notepad-plus-plus#10950]
### Description of the Issue
I accidently mash an unknown key combination and a yellow dialog containing what appears to be the full filenames of the opened tabs.  This can't be dismissed, I'd have expected the **{ESC}** key to work but nothing else I have tried works either.

In the mean time the editor is unusable until I click one of the items

### Expected Behavior
Anything that can be opened should be able to be dismissed with at a minimum the standard "escape" key.

### Debug Information
Notepad++ v8.1.9.3   (64-bit)
Build time : Dec  6 2021 - 19:21:37
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "D:\DBAREIS\FamilyTree\FAG (cemetery visits)\Vic\Dandenong, Bunurong Memorial Park (very pretty place with lake)\Mervyn Sidney Aubert.GenealogyResearch"
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro N (64-bit) 
OS Version : 2009
OS Build : 19044.1415
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 

![Screenshot @ 26_12_2021 , 4_38_37 PM,  +](https://user-images.githubusercontent.com/295230/147400059-5262389b-39f8-441c-9253-2d8c7b2d88e3.png)

**Comment by rdipardo** [comment]
That's the "Document Switcher". It appears when you press <kbd>CTRL</kbd> + <kbd>TAB</kbd> and have at least 2 files open. It should disappear when you release the keys.

You can also disable it under `Edit > Preferences > MISC.`:

![doc-switcher](https://user-images.githubusercontent.com/59004801/147407504-15952756-28a1-40dc-8d34-30fcebb15374.png)

Perhaps an empty check box would be a better default setting . . . ?

**Comment by alankilborn** [comment]
>I accidently mash an unknown key combination and a yellow dialog containing what appears to be the full filenames of the opened tabs

You pressed Ctrl+Tab (or, less likely, Ctrl+Shift+Tab).  This activates the "doc switcher" functionality.

Normal use is to hold Ctrl and tap and release Tab.  This will cycle the highlighting in the "yellow dialog" between your tabs.  When you finally release Ctrl, Notepad++ should switch to the highlighted file.

>This can't be dismissed

I had this trouble myself a long time ago, so I disabled the feature.  It is interesting that it causes problems for certain individuals, it would be nice to "root cause" they why of it and fix it for everyone.

**Comment by dbareis** [comment]
Thanks I will disable this.  

I actually use CTRL+SHIFT+Q, **I can reproduce the bug** if I hit between the TAB & Q keys (press both at once).

Either way it should be able to be dismissed by an ESC key.

**Comment by alankilborn** [comment]
> I can reproduce the bug if I hit between the TAB & Q keys (press both at once)

There is no "both at once" when talking about keys such as Tab and q.  One will always be before the other.  If it is Tab it will invoke the Doc Switcher functionality.

>it should be able to be dismissed by an ESC key.

Well, it should auto-close when the final key held is released.  Apparently it DOES auto-close for most people, it is just the small number of people where it DOESN'T auto-close makes it a problem.

**Comment by ArkadiuszMichalski** [comment]
Maybe same happend as here https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10574.

**Comment by alankilborn** [comment]
@ArkadiuszMichalski Well not the whole issue, but I think mere-human's comment there has it:  https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10574#issuecomment-923205006

**Comment by dbareis** [comment]
@alankilborn 

You appear to have missed the fact that I can reproduce the issue pretty much at will.  I described how to reproduce it, obviously there are milliseconds involved either way!

Obviously its not effecting people, its effected by conditions or user actions, the fact that a bug doesn't occur very often doesn't convert it from a bug to something else.  Programs need to work in and react to real life conditions.  There is a reason for testing, in an ideal world no testing would ever be required because the author knows everything.

It doesn't auto close for many, apparently Ctrl+F4. can be used to execx. I don't why why the standard ALT+F4 + ESC wouldn't be chosen for that functionality.  Key combinations don't need to be made up where standards exist.

**Comment by alankilborn** [comment]
>You appear to have missed the fact that I can reproduce the issue pretty much at will

No, didn't miss that...not sure why you think so.  Actually, not sure what your entire last post is really saying...sorry.

**Comment by sbc279** [comment]
I have the same issue at times, however it's usually triggered by me accidentally pressing the center "wheel"  button on the mouse. It would be nice to ESC/abort out of it.

**Comment by rimrul** [comment]
The worst part about triggering it through the mouse wheel is that when it happens I'm usually scrolling fast through a long document. This causes the dialog to start scrolling fast through my long list of documents as soon as it opens. That in turn doesn't exactly help me find back to the document I was looking at.

**Comment by CennoxX** [comment]
> The worst part about triggering it through the mouse wheel is that when it happens I'm usually scrolling fast through a long document. This causes the dialog to start scrolling fast through my long list of documents as soon as it opens. That in turn doesn't exactly help me find back to the document I was looking at.

You can use Ctrl+Tab to go back, if "Enable MRU behaviour" is activated.

**Comment by dbareis** [comment]
There might be any number of magic ways to get back once you know about it about it and happens often enough that you remember it that information but something should be done about this so the 99.65% of people who get stuck, don't.  At a minimum the "esc" key should work but better would be a button to exit.

**Comment by Kane-sigCare** [comment]
Having 'ESC' cancel the menu would be great. So many times I accidentally right-click when scrolling fast and open that menu. The feature is good, so I don't want to disable it.
